Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.23@sha256:7ea4c9dcb2b97ff8ee80a67db3d44f98c8ffa0d191399197007d8459c1453041 AS builder
```

Comment:
I am curious is it possible to set a tag (i.e. 1.23.4) instead of a digest? cc @jmhbnz @hakman @soma00333

Reply:
Agreed - Raised https://github.com/etcd-io/etcd-operator/pull/39 to supercede this pr.

Reply:
thanks